### PR TITLE
add yoto_summary

### PR DESCRIPTION
**Pull request by praeclarumjj3**

### CODE REVIEW

**Comment by aniket-agarwal1999** [comment]
PS: The idea seems pretty great and intuitive. Will it even work for some of the more harder tasks? Maybe you can once look into that and add it up as an disadvantage/advantage in your two cents section accordingly :)

File: summaries/You_only_train_once.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+- The conceptual simplicity of this approach makes it applicable to many problem domains, with only minimal changes to existing code bases.
+
+- The experiments have been carried out on three problems: Beta-VAE, Style Transfer and Learned Image Compression. The results showed close similarity with fixed-weight models trained for each of the parameter configurations.  
+
+## Opinion
+
+- YOTO could reduce the effort spend in training models with fixed parameters separately.
+
+- YOTO could turn out to be beneficial for many generative models where we have to keep in mind many different factors while training.
```
[the remark this answers is not in the document]
Comment (praeclarumjj3) [reply summaries/You_only_train_once.md]: Thanks for the quick reply! I have made changes accordingly. Please review them :)

**Comment by aniket-agarwal1999** [comment]
Add up the paper also in the README in the same format as the other papers are mentioned.

**Comment by praeclarumjj3** [comment]
> Add up the paper also in the README in the same format as the other papers are mentioned.

Done!

**Comment by aniket-agarwal1999** [comment]
Cool merging it now :)

**Pull request merged**